config/schema.yaml

```diff
@@ -8,4 +8,8 @@ properties:
                 type: number
                 enum: [2006, 2010, 2013, 2016, 2021]
                 default: 2016
-                description: Indicates the reference NUTS year
\ No newline at end of file
+                description: Indicates the reference NUTS year
+            wdpa-date:
+                type: string
+                enum: [Mar2021, Apr2021]
```

Comment:
I would not restrict the value in this way as this will make the schema invalid over time. Users will have to touch two places to update. I'd make this a string type instead.